packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -184,6 +186,9 @@ class _DismissibleState extends State<Dismissible> with TickerProviderStateMixin
   Size _sizePriorToCollapse;
 
   @override
+  bool get wantKeepAlive => _moveController?.isAnimating == true || _resizeController?.isAnimating == true;
```

Comment:
This seems like a common pattern (wanting to keep something alive because of running animations).  I wonder if we want a mixin that helps with that specifically.  You could just register your animations with it, and it could register the status listeners / read isAnimating itself.